src/mission_ui.cpp

```diff
@@ -71,9 +71,13 @@ class mission_ui_impl : public cataimgui::window
         mission_ui_tab_enum selected_tab = mission_ui_tab_enum::ACTIVE;
         mission_ui_tab_enum switch_tab = mission_ui_tab_enum::num_tabs;
 
-        size_t window_width = str_width_to_pixels( TERMX ) / 2;
-        size_t window_height = str_height_to_pixels( TERMY ) / 2;
-        size_t table_column_width = window_width / 2;
+        float window_width = std::clamp( float( str_width_to_pixels( 80 ) ),
```

Comment:
This should use `EVEN_MINIMUM_TERM_WIDTH` from game_constants.h.

The height should use `EVEN_MINIMUM_TERM_HEIGHT`.

We already have these defined as constants, so no additional work is needed besides using the existing constexpr.

Reply:
Implemented and tested. 

Sorry for wasting everyone's time on something that's supposed to be simple..  
I've looked at most of the UI code while during developing my fork, and the execution of each window is so different, I don't know what's considered standard.

Reply:
You're not wasting anybody's time.